File: minwook/4주차/중복된_숫자_개수.java
class Solution {
    public int solution(int[] array, int n) {
        int answer = 0;
        for(int i : array){
            if(i==n) // array에 n과 같은 숫자가 있으면 answer 값 +1
                answer++;
        }
        return answer;
    }
}

File: yongkeun/4week/중복된숫자개수.js
function solution(array, n) {
  var answer = 0;
  array.forEach((value) => {
    //array 배열을 돌며,
    if (value === n) {
      //값이 같으면
      answer++; //더하기
    }
  });
  return answer;
}
